app/views/static_pages/sponsors.html.haml

```diff
@@ -0,0 +1,18 @@
+.col-xs-8.col-xs-offset-2
+  %h1= t('.title')
+
+  %hr
+
+  %h3 Many thanks to our sponors!
+
+.col-xs-8.col-xs-offset-2
+  - @sponsors.each do |sponsor|
+    .row.sponsor-row
+      .col-xs-5
+        %div{class: t("sponsors.#{sponsor}.class_name")}
+          %span.l-text-hidden t("sponsors.#{sponsor}.name")
+      .col-xs-7
+        %h3
+          = t("sponsors.#{sponsor}.name")
+        = link_to t("sponsors.#{sponsor}.link"), target: '_blank' do
```

Comment:
I think we should make the whole `div` for each sponsor (logo, title text) a link. It isn't immediately clear to me that the description leads to the sponsor's page. :smile_cat: